README.md

```diff
@@ -5,17 +5,30 @@ This action uses [__flake8__](http://flake8.pycqa.org) to know if your Python co
 
 If syntax errors are found then this Action uses [__futurize__](http://python-future.org/futurize_cheatsheet.html) create pull requests that gradually upgrade that code to be more compatible with Python 3.  After this Action has run, return to your repo and look for a "__modernize-Python-2-codes__" branch in your repo.  If this branch exists then select it and __make pull request__ and make sure that your automate tests pass before merging the pull request.  After merging, delete the "__modernize-Python-2-codes__" branch so that the process can be repeated.
 
-Example workflow (Put the following text into your repo's `.github/main.workflow`):
+Example workflow (Put the following text into your repo's `.github/workflows/py3.yml`):
 ```
-workflow "New workflow" {
-  on = "push"
-  resolves = ["Upgrade to Python 3"]
-}
+on:
+  push:
+    branches:
+      - py3-upgrade
```

Comment:
upgrade_to_Python3

Reply:
one thing to bear in mind is this would be user specifyable, its the branch that you want to act on. for example i didn't want every push to master initiating the py3 thing, it was only my local py3 branch